viral/fisher.py

```
@@ -0,0 +1,443 @@
+            yerr=[
+                df_sub[location].std() / np.sqrt(len(df_sub))
+                for location in location_names_plot
+            ],
+        )
+        plt.ylabel("Fraction of neurons")
+        plt.title(f"{genotype}")
+        plt.ylim(0, 0.6)
+        plt.tight_layout()
+        plt.savefig(CACHE_PATH / f"{genotype}_{stage}_binary_peak.png")
+
+    1 / 0
```

Ruff (0.15.5)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)